tests/ArduinoStringTest/scripts/run_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -u
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(dirname "$SCRIPT_DIR")"
+BUILD_DIR="$PROJECT_ROOT/build/Release"
+EXECUTABLE="$BUILD_DIR/ArduinoStringTest"
+
+if [[ ! -x "$EXECUTABLE" ]]; then
```

Comment:
When we are automating this with CI, the flow should be
1. run_test calls cmake ot create the executable as well.
2. Then runs the executable and compares the result.